doc/release_notes.rst

```diff
@@ -21,9 +21,19 @@
 
 .. contents::
 
-===================
-2.0.0 Release Notes
-===================
+=======================
+2.0.0-rc2 Release Notes
```

Comment:
I think that writing release notes for downstream only releases (2.0.0-rc2 has not been tagged in master) might be confusing for external contributors and users.